docs/architecture/adr-023-data-fee-market.md

```diff
@@ -0,0 +1,60 @@
+- **Bounded Supply**: The supply of block space can be seen as a bounded commodity. At a point, buyers begin to compete for space as they hit the upper bound on what the network can provide. Crossing this line could result in price wars with rollups bidding against each other. Before that point, all data could be published so long as they are seen by the validators to cover the costs. Improvements in scaling of the protocol can naturally shift that upper bound.
+- **Different Denominations**: The denomination of the transaction between the buyer and seller, TIA, differs to the denomination of the costs that the seller incurs (i.e. USD, EUR which server providers like AWS or Hetzner charge in). The buyer may also need to bridge price differences between TIA and the currency that they receive from their users. As the relative prices may fluctuate due to market forces beyond the purview of the Celestia network, Celestia in turn may incorrectly price the cost of publishing data.
+- **Secondary Markets and MEV**: If Celestia is able to offer value to it's users but is not able to correctly capture that value, it risks being captured on secondary markets that auction block space off chain and use other currencies. Moving payment offchain reduces the utility of the TIA token that secures the network. Actors such as validators who wish to extract as much value as possible (MEV) may make selfish decisions that harm the integrity of the network as a whole.
+- **Price Elasticity and Competition**: Price elasticity refers to how much a buyers demand for a commodity changes as the price changes. Given a Rollups strong dependence on Celestia for liveness (their service stops if they are unable to submit data), the commodity is rather inelastic. The eventual increase in competition and fallback methods that allow a rollup to switch providers may reduce the elasticity. Inelastic goods generally gives greater power to the seller.
+
+## Decision
+
+Celestia's fee market should fall between the bounds of being cost-effective for both suppliers (validators) and buyers (rollups). It should endeavour to provide the greatest price certainty and use the resource of the network in the most efficient manner. To this extent the following decisions were made:
+
+- **Global Min Gas Price**: A new block validity rule has been introduced constraining the minimum gas price (in other words the fee divided by the gas limit) that a transaction can have. This is designed to mitigate the ability for payment to be settled off-chain. It also resolves the incongruence of nodes with different minimum limits. This value can be modified via on-chain governance. This therefore means there is a dependency on the community interested in the value of TIA to ensure that a minimum price does not lead to an uncompetitive price.
```

Comment:
[optional] the tense here makes it sound like a block validity rule has already been added but I don't think that's the case yet.

Reply:
Technically ADRs should be written in the past tense because it is supposed to document a decision that has happened however since we use the same format as proposals people often write with future tense.

Reply:
Ahh that's confusing. Down to adopt either past or future tense as long as ADRs are consistent.

My fuzzy feeling is that ADRs should use future tense when in the Proposed state and then past tense if moved from Proposed to Accepted. If the ADR never gets accepted, it can remain in future tense. But that may be too much overhead.

Reply:
Yeah I get that although it might be cumbersome to have to go through and change the tense (unless chatGPT can do that for us)